sorting/count_sort/count_sort.py

```diff
@@ -0,0 +1,36 @@
+"""
+Logic to handle negative numbers in counting sort was referred from the below link:
+https://www.ripublication.com/ijaer18/ijaerv13n1_28.pdf
+"""
+def counting_sort(L):
+    if not L:
+        return L
+    largest = max(L)
+    min_value = min(L)
+    offset = -min_value if min_value<0 else 0
+    n = len(L)
+    frequencyArray = [0]*(largest+1+offset)
+    for i in range(n):
+     # shift of the element  by the offset which ensures negative numbers are shifted to non-negative indices in frequencyArray
+        frequencyArray[L[i]+offset] = frequencyArray[L[i]+offset]+1
+    #sorting part

```

Comment:
Usually a space is left after a comment token and the upper and lower cases are followed exactly like English. And followed by a full stop in the end.
`# Sorting part.`

Update it here as well as places below.